base/operators.jl

```diff
@@ -1201,6 +1201,19 @@ end
 (f::Fix{1})(arg; kws...) = f.f(f.x, arg; kws...)
 (f::Fix{2})(arg; kws...) = f.f(arg, f.x; kws...)
 
+function Base.show(io::IO, fix::Fix{N}) where {N}
+    constr = Fix{N}
+    callable = fix.f
+    fixed_argument = fix.x
+    show(io, constr)
+    print(io, '(')
+    show(io, callable)
+    print(io, ',')
+    print(io, ' ')
+    show(io, fixed_argument)
+    print(io, ')')
+end
+
```

Comment:
Might it be cleaner just to redirect from `show(::Function)` back to show_default, so we don't need to write out an explicit hand-written copy of `show_default`, and also perhaps to define `showarg`?
```suggestion
show(io::IO, fix::Fix) = show_default(io, fix)
showarg(io::IO, fix::Fix{N}, toplevel) where {N} = (print(io, "Fix{", N, "}("); showarg(io, fix.f, false); showarg(io, fix.x, false); print(io, ")")
```